dev-requirements.txt

```diff
@@ -37,6 +37,7 @@ pytest-cov>=2.4.0,<2.6
 Sphinx>=1.7.6
 sphinx-git>=10.1.1
 GitPython>=2.1.1;
+sphinxcontrib-fulltoc>=1.2.0
```

Comment:
This new dependency also needs to be added to `minimum-constraints.txt` as:
```
sphinxcontrib-fulltoc==1.2.0
```

Reply:
DONE